.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -199,6 +199,21 @@ jobs:
           retention-days: 15
           if-no-files-found: error
 
+  # We don't need the binaries installed in the rootfs as part of the tarball, so can delete them now we've built the rootfs
+  remove-rootfs-binary-artifacts:
+    runs-on: ubuntu-22.04
+    needs: build-asset-rootfs
+    strategy:
+      matrix:
+        asset:
+          - agent
+          - coco-guest-components
+          - pause-image
```

Comment:
Hmm, good point. I guess making our release tarball and test match as much as possible makes sense, so that would mean we should clean it up (which would only do anything on the non-release stage)?

Reply:
> Hmm, good point. I guess making our release tarball and test match as much as possible makes sense, so that would mean we should clean it up (which would only do anything on the non-release stage)?

Yep, but I'm not sure if that's worth the trouble.  For now, leave it as it is, please. :-)